CMakeLists.txt

```diff
@@ -17,29 +17,17 @@ else()
 endif()
 ###############################################################################
 
-find_package(ZED 1.0 REQUIRED)
-
-##For Jetson, OpenCV4Tegra is based on OpenCV2.4
-exec_program(uname ARGS -p OUTPUT_VARIABLE CMAKE_SYSTEM_NAME2)
-if ( CMAKE_SYSTEM_NAME2 MATCHES "aarch64" OR CMAKE_SYSTEM_NAME2 MATCHES "armv7l" )
-    SET(OCV_VERSION "2.4")
-    SET(CUDA_VERSION "")
-else()
-    SET(CUDA_VERSION "7.5")
-    SET(OCV_VERSION "3.1")
-endif()
-
-find_package(OpenCV ${OCV_VERSION} COMPONENTS core highgui imgproc REQUIRED)
-find_package(CUDA ${CUDA_VERSION} REQUIRED)
+find_package(ZED REQUIRED)
 
+find_package(CUDA REQUIRED)
+find_package(OpenCV 3.1 EXACT REQUIRED COMPONENTS core highgui imgproc REQUIRED)
```

Comment:
I don't think this is a good idea to hard replace. Platforms such as the Jetson TX1 (and TK1) often employ opencv4tegra, which is based off opencv 2.4.X. Either leave it as is, or make a case where it can handle both?